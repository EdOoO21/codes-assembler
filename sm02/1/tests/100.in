-59 -69 949 578 -643 444 -196 499 -284 -113 38 638 -775 91 -752 -836 508 -67
