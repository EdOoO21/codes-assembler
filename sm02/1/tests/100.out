-67
508
-836
-752
91
-775
638
38
-113
-284
499
-196
444
-643
578
949
-69
-59
